src/app/charts/chart.component.html

```diff
@@ -11,6 +11,12 @@ <h2 class="chart-label">
         </button>
     </div>
 </div>
+<div *ngIf="chart.indicator.thresholdIndicator" class="chart-options">
```

Comment:
Not sure. We can leave as is and evaluate that in one of the later PRs that adds the additional components.